src/SipConnector.ts

```diff
@@ -937,8 +950,16 @@ export default class SipConnector {
     });
   };
 
-  _generateStream(videoTrack, audioTrack) {
-    const { id } = videoTrack;
+  _generateStream(
+    videoTrack: MediaStreamTrack,
+    audioTrack?: MediaStreamTrack
+  ): MediaStream | undefined {
+    const id = videoTrack?.id || audioTrack?.id;
```

Comment:
why this changed?

Reply:
restored (was changed for creating audio stream with no video track)